tests/engines/registries/acceptance/overview/overview-test.ts

```diff
@@ -0,0 +1,110 @@
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { TestContext } from 'ember-test-helpers';
+import moment from 'moment';
+import { module, test } from 'qunit';
+
+import config from 'ember-get-config';
+import RegistrationModel from 'ember-osf-web/models/registration';
+import { visit } from 'ember-osf-web/tests/helpers';
+import { setupEngineApplicationTest } from 'ember-osf-web/tests/helpers/engines';
+import pathJoin from 'ember-osf-web/utils/path-join';
+
+const tagsMap: Record<string, string[]> = {
+    title: ['dc.title', 'citation_title', 'og:title', 'twitter:title'],
+    description: ['dc.abstract', 'citation_description', 'og:description', 'twitter:description'],
+    contributors: ['dc.creator', 'citation_author'],
+    affiliatedInstitutions: ['citation_author_institution'],
+    tags: ['dc.keywords'],
+    dateRegistered: ['dc.datesubmitted', 'citation_publication_date'],
+    dateModified: ['dc.datemodified'],
+    url: ['citation_public_url', 'og:url'],
+    image: ['og:image', 'twitter:image'],
+};
+
+function getMetaTagsFor(assert: Assert, prop: string): Array<string|null> {
+    const terms = tagsMap[prop];
+
+    const metaTags = terms.map(term => {
+        const k = term.includes('og:') ? 'property' : 'name';
+        const selector = `meta[${k}="${term}"]`;
+        const elements = [...document.querySelectorAll(selector)];
+        return elements.map((el: Element) => el.getAttribute('content'));
+    });
+
+    if (metaTags.length > 1) {
+        assert.ok(metaTags.every(m => JSON.stringify(m) === JSON.stringify(metaTags[0])));
+    }
+
+    return metaTags[0];
```

Comment:
It might tidy things up if this function also took an expected value and asserted against it, instead of returning:
```ts
function assertMetaTags(assert: Assert, prop: string, expected: string | string[]): void {
  // ...
  const expectedArray = typeof expected === 'array' ? expected : [expected];
  assert.deepEqual(expectedArray, metaTags[0], `Registration.${prop} does not match...`);
}
```
The body of the test becomes more like:
```ts
assertMetaTags(assert, 'title', reg.title);
assertMetaTags(assert, 'description', reg.description);
assertMetaTags(assert, 'url', pathJoin(config.OSF.url, registration.id));
assertMetaTags(assert, 'contributors', contributors.mapBy('users.fullName'));
// etc.
```